src/main/resources/META-INF/rewrite/was-to-liberty.yml

```diff
@@ -34,6 +34,7 @@ recipeList:
   - org.openrewrite.xml.liberty.WebDDNamespaceRule
   - org.openrewrite.xml.liberty.DetectUnsupportedJSF
   - org.openrewrite.maven.liberty.AddOpenLibertyPlugin
+  - org.openrewrite.maven.WasDevMvnChangeParentArtifactId
```

Comment:
Should we change the package here and below to match `org.openrewrite.maven.liberty` used just above?

Reply:
```suggestion
  - org.openrewrite.maven.liberty.WasDevMvnChangeParentArtifactId
```